web-marketplace/src/components/organisms/CreateSellOrderForm/CreateSellOrderForm.tsx

```diff
@@ -139,11 +145,15 @@ const CreateSellOrderForm: React.FC<Props> = ({
       </Box>
       <AmountField
         label="Amount to sell"
-        availableAmount={availableAmountByBatch[batchDenom ?? '']}
+        availableAmount={availableAmount}
         error={!!errors['amount']}
         helperText={errors['amount']?.message}
         denom={batchDenom ?? ''}
-        customInputProps={{ step: 'any' }}
+        customInputProps={{
+          step: 'any',
+          max: availableAmount,
```

Comment:
should this be directly in `AmountTextField` component so we don't have to repeat it every time we use `AmountField`?

Reply:
I guess we can yes since this field is meant to be used with max set. We can always override that with prop if we need to.